hq/app/aws/AWS.scala

```diff
@@ -52,37 +52,40 @@ object AWS {
     } yield AwsClient(client, account, region)
   }
 
-  def dynamoDbClient(securityCredentialsProvider: AWSCredentialsProvider, region: Regions, stage: Stage): AmazonDynamoDB = {
```

Comment:
I just moved a few methods around so the logic in this file was easier to follow (`clients` next to where it was called from, for example)